packages/shortest/src/browser/core/browser-tool.ts

```diff
@@ -201,24 +206,58 @@ export class BrowserTool extends BaseBrowserTool {
     await this.page.click(selector);
   }
 
-  public async clickAtCoordinates(x: number, y: number): Promise<void> {
-    this.log.debug("Clicking at coordinates", { x, y });
-    await actions.click(this.page, x, y);
-  }
-
   async execute(input: ActionInput): Promise<ToolResult> {
     try {
       this.log.setGroup(`🛠️ ${input.action}`);
       let output = "";
       let metadata = {};
 
       switch (input.action) {
-        case "left_click":
-        case "right_click":
-        case "middle_click":
-        case "double_click": {
```

Comment:
These didn't seem to work correctly before, as the actual button (`middle`, etc), or the number of clicks, were not passed to Playwright.